kubernetes/aks/apps/demo/otel/base/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: nginx-ingress
+  namespace: nginx
+  annotations:
+    external-dns.alpha.kubernetes.io/target: inspection.alpha.canada.ca
```

Comment:
Maybe whitelist the VPN or put a vouch in front of the application? So that it is not accessible to everyone.

Reply:
Also, naming of the ingress should probably be `otel-ingress` with the namespace `monitoring`

Reply:
done